docs_src/microservices/device/Ch-DeviceServiceList.md

```diff
@@ -13,10 +13,9 @@ The following table lists the EdgeX device services and protocols they support.
 | [device-mqtt-go]( https://github.com/edgexfoundry/device-mqtt-go/tree/{{version}})           | MQTT | Active |  Two way communications via multiple MQTT topics | |
 | [device-modbus-go]( https://github.com/edgexfoundry/device-modbus-go/tree/{{version}})       | Modbus | Active | Supports Modbus over TCP or RTU | |
 | [device-gpio]( https://github.com/edgexfoundry/device-gpio/tree/{{version}})                 | GPIO | Active | Linux only; uses sysfs ABI | |
-| [device-grove-c](https://github.com/edgexfoundry/device-grove-c/tree/v1.3.1)                                 | | **2.x TBD** | Connects the Grove sensor on Grove Raspberry Pi using libmraa library; Linux and ARM only | |
-| [device-bacnet-c]( https://github.com/edgexfoundry/device-bacnet-c/tree/v1.3.1)                              | BACnet | **2.x TBD** | Supports BACnet via ethernet (IP) or serial (MSTP).  Uses the Steve Karag BACnet stack | |
-| [device-coap-c]( https://github.com/edgexfoundry/device-coap-c/tree/{{version}})                                  | CoAP | Active (**2.2.0 TBD**) | This service is in the process of being redeveloped and expanded for upcoming release for Kamakura – and will support Thread as a subset of functionality.  Currently supports CoAP-based REST and is one way communications (read-only) | |
-| [device-uart]( https://github.com/edgexfoundry-holding/device-uart)                                          | UART | **in Development** | Linux only; for connecting serial UART devices to EdgeX | |
+| [device-bacnet-c]( https://github.com/edgexfoundry/device-bacnet-c/tree/v1.3.1)                              | BACnet | Active | Supports BACnet via ethernet (IP) or serial (MSTP).  Uses the Steve Karag BACnet stack | |
```

Comment:
device-bacnet-c has been upgraded to 3.0
https://github.com/edgexfoundry/device-bacnet-c/tree/v3.0.0

Use `https://github.com/edgexfoundry/device-bacnet-c/tree/{{3.0.0}}` as well